lib/Config/MP2Migrator.php

```diff
@@ -15,6 +15,15 @@
 
 if(!defined('ABSPATH')) exit;
 
+/**
+ * @property string $mp2_campaign_table
+ * @property string $mp2_custom_field_table
+ * @property string $mp2_email_table
+ * @property string $mp2_form_table
+ * @property string $mp2_list_table
+ * @property string $mp2_user_table
+ * @property string $mp2_user_list_table
+ */
```

Comment:
Why not putting them as real public properties instead of being annotated here?

Reply:
I just probably understood this during the Premium review: https://github.com/mailpoet/mailpoet-premium/pull/165#discussion_r250596658

I guess anything is OK for now since we don't have this generally unified.

Reply:
Good point, will update it.